Makefile.PL

```diff
@@ -2,17 +2,28 @@ use ExtUtils::MakeMaker;
 
 # Use: perl Makefile.PL OPTIMIZE="-O0 -g -Wdeclaration-after-statement"
 
-my $yazconf = "yaz-config";
-my $yazver = `$yazconf --version`;
-my $yazinc = `$yazconf --cflags servers`;
-my $yazlibs = `$yazconf --libs server`;
-if (!$yazver || (!$yazinc && !$yazlibs)) {
-   die qq[
+my $yazver;
+my $yazinc;
+my $yazlibs;
+system("pkg-config --exists yaz-server");
+if ($? == 0) {
+    $yazconf = "pkg-config";
+    $yazver = `pkg-config --modversion yaz-server` or die;
```

Comment:
If you're going to the trouble of defining `$yazconf` you should probably use it here and in the next two lines. On the other hand, why not get rid of it completely, both here and in the alternative clause? It doesn't really buy us anything.

Reply:
Sure.